### PR TITLE
[3.1] Update nodeos options help usage [docs]

### PR DESCRIPTION
**Pull request by lparisc**
Resolves eosnetworkfoundation/devrel#60

### CODE REVIEW

File: docs/01_nodeos/03_plugins/producer_plugin/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Config Options for eosio::producer_plugin:
                                         <public-key>=<provider-spec>
                                         Where:
                                            <public-key>    is a string form of
-                                                           a vaild Antelope public
+                                                           a vaild EOSIO public
```
Comment (iamveritas) [inline docs/01_nodeos/03_plugins/producer_plugin/index.md:52]: EOSIO -> EOS

File: docs/01_nodeos/03_plugins/producer_plugin/index.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Config Options for eosio::producer_plugin:
                                            <provider-type> is KEY, or KEOSD
 
                                            KEY:<data>      is a string form of
-                                                           a valid Antelope
+                                                           a valid EOSIO
```
Comment (iamveritas) [inline docs/01_nodeos/03_plugins/producer_plugin/index.md:62]: EOSIO -> EOS

**Review by iamveritas — CHANGES_REQUESTED** [review]
EOSIO needs to be changed

**Comment by lparisc** [comment]
Redoing due to EOL character issues.

File: docs/01_nodeos/03_plugins/net_plugin/index.md [hunk 3]
```diff
@@ -35,16 +35,16 @@ Config Options for eosio::net_plugin:
                                         indicates to node that only
                                         transactions 'trx' or blocks 'blk'
                                         should be sent.  Examples:
-                                            p2p.example.org:9876
-                                            p2p.trx.example.org:9876:trx
-                                            p2p.blk.example.org:9876:blk
+                                            p2p.eos.io:9876
+                                            p2p.trx.eos.io:9876:trx
+                                            p2p.blk.eos.io:9876:blk
```
Comment (ericpassmore) [inline docs/01_nodeos/03_plugins/net_plugin/index.md:40]: I don't have strong feelings here. [Previously this was changed ](https://github.com/eosnetworkfoundation/mandel/pull/748/files#diff-5cd6d2d6d53259f0b62b7e22b7baf9951b24232907c8d4bb5ea17929199206f5) to example.org to comply with [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.txt) see 2.

Comment (matthewdarwin) [reply docs/01_nodeos/03_plugins/net_plugin/index.md:40]: If the *.eos.io is not a real endpoing, then better to use example.org.

Comment (lparisc) [reply docs/01_nodeos/03_plugins/net_plugin/index.md:40]: changed to example.org and created alternate PR #583

**Pull request closed without merging**